test/unit/org/apache/cassandra/tools/nodetool/GcStatsTest.java

```diff
@@ -0,0 +1,144 @@
+import static org.assertj.core.api.Assertions.assertThatCode;
+
+public class GcStatsTest extends CQLTester
+{
+    @BeforeClass
+    public static void setUp() throws Exception
+    {
+        requireNetwork();
+        startJMXServer();
+    }
+
+    @Test
+    @SuppressWarnings("SingleCharacterStringConcatenation")
+    public void testMaybeChangeDocs()
+    {
+        // If you added, modified options or help, please update docs if necessary
+        ToolRunner.ToolResult tool = ToolRunner.invokeNodetool("help", "gcstats");
+        tool.assertOnCleanExit();
+
+        String help =   "NAME\n" +
+                        "        nodetool gcstats - Print GC Statistics\n" +
+                        "\n" +
+                        "SYNOPSIS\n" +
+                        "        nodetool [(-h <host> | --host <host>)] [(-p <port> | --port <port>)]\n" +
+                        "                [(-pp | --print-port)] [(-pw <password> | --password <password>)]\n" +
+                        "                [(-pwf <passwordFilePath> | --password-file <passwordFilePath>)]\n" +
+                        "                [(-u <username> | --username <username>)] gcstats\n" +
+                        "                [(-F <format> | --format <format>)]\n" +
+                        "\n" +
+                        "OPTIONS\n" +
+                        "        -F <format>, --format <format>\n" +
+                        "            Output format (json, yaml)\n" +
+                        "\n" +
+                        "        -h <host>, --host <host>\n" +
+                        "            Node hostname or ip address\n" +
+                        "\n" +
+                        "        -p <port>, --port <port>\n" +
+                        "            Remote jmx agent port number\n" +
+                        "\n" +
+                        "        -pp, --print-port\n" +
+                        "            Operate in 4.0 mode with hosts disambiguated by port number\n" +
+                        "\n" +
+                        "        -pw <password>, --password <password>\n" +
+                        "            Remote jmx agent password\n" +
+                        "\n" +
+                        "        -pwf <passwordFilePath>, --password-file <passwordFilePath>\n" +
+                        "            Path to the JMX password file\n" +
+                        "\n" +
+                        "        -u <username>, --username <username>\n" +
+                        "            Remote jmx agent username\n" +
+                        "\n";
+
+        assertThat(tool.getStdout().trim()).isEqualTo(help.trim());
+    }
+
+    @Test
+    public void testDefaultGcStatsOutput()
+    {
+        ToolRunner.ToolResult tool = ToolRunner.invokeNodetool("gcstats");
+        tool.assertOnCleanExit();
+        String output = tool.getStdout();
+        assertThat(output.contains("Interval (ms)"));
```

Comment:
I think it should be either `assertThat(output).contains("Interval (ms)");` or `assertTrue(output.contains("Interval (ms)"));`